redisdb/datadog_checks/redisdb/data/conf.yaml.example

```diff
@@ -1,84 +1,128 @@
 init_config:
 
 instances:
+
+  ## @param host - string - required
+  ## Enter the host to connect to.
+  #
   - host: localhost
+
+  ## @param port - integer - required
+  ## Enter the port of the host to connect to.
+  #
     port: 6379
 
-    # Can be used in lieu of host/port
-    #
-    # unix_socket_path: /var/run/redis/redis.sock # optional, can be used in lieu of host/port
-
-    # Addional connection options
-    #
-    # db: 0
-    # password: mypassword
-    # socket_timeout: 5
-
-    # Optional SSL/TLS parameters
-    # ssl: False     # Optional (default to False)
-    # ssl_keyfile:   # Path to the client-side private keyfile
-    # ssl_certfile:  # Path to the client-side certificate file
-    # ssl_ca_certs:  #  Path to the ca_certs file
-    # ssl_cert_reqs: # Specifies whether a certificate is required from the
-    #                # other side of the connection, and whether it will be validated if
-    #                # provided.
-    #   * 0 for ssl.CERT_NONE (certificates ignored)
-    #   * 1 for ssl.CERT_OPTIONAL (not required, but validated if provided)
-    #   * 2 for ssl.CERT_REQUIRED (required and validated)
-
-    # Optional extra tags added to all redis metrics
-    # tags:
-    #   - optional_tag1
-    #   - optional_tag2
-    #
-
-    # Collect the lengths of the following keys.
-    # Length is 1 for strings
-    # Length is zero for keys that have a type other than list, set, hash, or sorted set.
-    # Keys can be expressed as patterns, see https://redis.io/commands/keys
-    #
-    # keys:
-    #   - key1
-    #   - key2
-    #   - key* (matches key, key1 and key2)
-
-    # If you provide a list of 'keys', have the Agent log a warning when keys are missing.
-    # (default: True)
-    #
-    # warn_on_missing_keys: True
-
-    # Max number of entries to fetch from the slow query log
-    # By default, the check will read this value from the redis config
-    # If it's above 128, it will default to 128 due to potential increased latency
-    # to retrieve more than 128 slowlog entries every 15 seconds
-    # If you need to get more entries from the slow query logs
-    # set the value here.
-    # Warning: It may impact the performance of your redis instance
-    # slowlog-max-len: 128
-
-    # Collect INFO COMMANDSTATS output as metrics.
-    # command_stats: False
-
-    # Disable connections cache.
-    # The check attempts to reuse the same Redis connections at every collection
-    # cycle, this behaviour can be disabled to prevent stale connections.
-    # disable_connection_cache: False
+  ## @param unix_socket_path - string - optional
+  ## Connect through a unix socket instead of
+  ## using a `host` and a `port`.
+  #
+  #  unix_socket_path: /var/run/redis/redis.sock
+
+  ## @param db - integer - optional - default: 0
+  ## XXXXXXXX
```

Comment:
What could be the definition of this parameter?

Reply:
Good catch! Here https://www.rforge.net/doc/packages/rediscc/redis.connect.html

Reply:
```suggestion
  ## Index of the database (keyspace) to use. 
  ## The index 0 is the default and any other index will result in a SELECT command to be sent upon connection
  ## to select the desired database. For redis.clone this can also be NA in which case the same database is 
  ## used as in rc.
```